WorkloadManagementSystem/Client/Matcher.py

```diff
@@ -334,7 +335,8 @@ def _checkCredentials(self, resourceDict, credDict):
             raise RuntimeError(result['Message'])
           if credDict['group'] not in result['Value']:
             # DN is not in the same group! bad boy.
-            self.log.notice("You cannot request jobs from DN %s. It does not belong to your group!" % ownerDN)
+            self.log.notice("You cannot request jobs from this DN, as it does not belong to your group!",
```

Comment:
Should this message really be `notice` ?

Reply:
`warn`?

Reply:
I don't know, I did not check the logic, but being forbidden to do something and being a naugthy boy is quite strong :p